### DISCUSSION TITLE
-ts scale Texture only on Texture Size

### DISCUSSION THREAD

**Discussion by arpu** [zeux/meshoptimizer#397] [Ideas]
Hey,

would be nice to have a option to scale Texture down if  width/height >= 4096 as example

or to have a option the max Texture is 2048

**Comment by zeux** [comment]
Yup that makes sense; this needs a command line argument to cap the max texture size.

**Reply by zeux** [reply]
Implemented in https://github.com/zeux/meshoptimizer/pull/398
